src/pages/Secret/Secret.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import {BrowserRouter} from 'react-router-dom';
+import AuthProvider from '../../providers/Auth';
+import Secret from './index';
+
+describe('Secret Components', () => {
+    test('should take a snapshot', () => {
+        const {asFragment} = render(
+            <AuthProvider>
+                <BrowserRouter>
+                    <Secret/>
+                </BrowserRouter>
+            </AuthProvider>
```

Comment:
I've seen this pattern repeated across several tests. While only testing to match the snapshot, would make sense to do it once and later build individual tests for the rest of the internal components. WDYT?